### DISCUSSION TITLE
public access of base [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2083] [Discord Threads]
> 🙋‍♂️ **jaan**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You could use share base feature  
#  
> 💁 **o1lab**  
#### Pls see this  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/share-base#generate-share-base  
#  
> 💁 **o1lab**  
#### Welcome to the community <@571762017195655188>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973730715143393289)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
